src/CompositeProducerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace HelloFresh\Reagieren;
+
+interface CompositeProducerInterface
```

Comment:
What's the reason for having this instead of just using the normal `ProducerInterface`?